Products/PloneKeywordManager/testing.py

```diff
@@ -27,6 +39,9 @@ def setUpPloneSite(self, portal):
         self.applyProfile(portal, 'Products.PloneKeywordManager:default')
         self.applyProfile(portal, 'plone.app.dexterity:default')
 
+        if PLONE_5:
```

Comment:
That will take a larger refactoring of the test code that I was ready to take on at this point @hvelarde.  Do you think this is a requirement for merging?